external_push/migrations/0008_brewfather updates.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-05-08 15:45
```

Comment:
Django 1.11? Boooo. :)

Upgrade to Django 3.0!

Reply:
Will do..

Reply:
I ended up recreating this migration under Django 3.0 /w the latest squashed commits. You may need to fake migrations in order to get this update to work on your install of Fermentrack (assuming you are running this branch in production)